src/Responses/CakeResponseFactory.php

```diff
@@ -2,16 +2,19 @@
 
 namespace League\Glide\Responses;
 
-use Cake\Network\Response;
+use Cake\Http\Response;
```

Comment:
Could we choose the Response to create depending on which is available (if class exists, with fallback on Cake\Http\Response)? This would allow us to support both Cake 3 and 4.

Reply:
Backwards compatibility with CakePHP version 3.x and 4.x

Reply:
use Cake\Network\Response; it is available until version 3.3. *, in versions above 3.4. * they are already another class use Cake\Http\Response;

https://github.com/cakephp/cakephp/blob/3.3.16/src/Network/Response.php

https://github.com/cakephp/cakephp/blob/3.4.0/src/Network/Response.php